include/evmc/helpers.hpp

```diff
@@ -101,6 +109,7 @@ template <>
 struct hash<evmc_address>
 {
     /// Hash operator using FNV1a.
+    EVMC_DEPRECATED
```

Comment:
Can't it be applied to the entire `struct hash<evmc_address>`?

Reply:
It can be.